alea/models/blueice_extended_model.py

```diff
@@ -153,25 +160,28 @@ def _build_ll_from_config(self, likelihood_config: dict) -> "LogLikelihoodSum":
                         "Only rate multipliers that end on _rate_multiplier"
                         " are currently supported.")
 
-                # Set shape parameters
-                shape_parameters = [
-                    p for p in source["parameters"] if self.parameters[p].ptype == "shape"]
-                if shape_parameters:
-                    # TODO: Implement setting shape parameters
-                    raise NotImplementedError("Shape parameters are not yet supported.")
-
                 # Set efficiency parameters
                 if source.get("apply_efficiency", False):
                     self._set_efficiency(source, ll)
 
+                # Set shape parameters
+                shape_parameters = [
+                    p for p in source["parameters"] if self.parameters[p].ptype == "shape"]
```

Comment:
This indent looks good to me. We can not find other ways to break the line. Also, the report of reviewdog might be a false positive, related to https://github.com/wemake-services/wemake-python-styleguide/issues/1416.